src/platform.hpp

```
@@ -38,7 +40,9 @@ typedef EbpfMapDescriptor& (*ebpf_get_map_descriptor_fn)(int map_fd);
 struct ebpf_platform_t {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: get_program_type, get_helper_prototype, get_helper_prototype_by_name, is_helper_usable, is_helper_usable_by_name, map_record_size, parse_maps_section, get_map_descriptor, get_map_type, resolve_inner_map_references, supported_conformance_groups

(cppcoreguidelines-pro-type-member-init)